src/ML/tests/MANNTest.cpp

```diff
@@ -81,17 +81,15 @@ TEST_CASE("MANN")
                                                   0.0, 1.3552527156068805e-20};
     input.baseVelocitiesTrajectory = Eigen::Map<Eigen::MatrixXd>(baseVelocitiesTemp.data(), 2, 12);
 
-    input.jointPositions << 0.14304762524836648, 0.031516238814005136, -0.1426330272419353, -0.06098283408307653,
-        -0.1341771055787397, 0.19911560020803554, 0.0328098631623236, 0.024781975683781033,
-        0.01324623650231824, 0.03895569208112638, -0.5063181281080337, -0.15188637555678797,
-        -0.09926268052516171, -0.35648960192960166, 0.8564996250726968,
-        -0.1773331496644323, -0.09618105207864167, -0.3637387390826281, -0.12775099875607665;
-
-    input.jointVelocities << -0.04892431424018461, 0.011816611105509, -0.030103574006318207, -0.054470913637429216,
-        0.04110477832027632, 0.006149116530581617, 0.0022679536303376294, -0.01772658684486485,
-        0.0018849445068382577, 0.0027670107680974943, 0.003149718526371251, 0.009372946811675064,
-        -0.019232750841138426, 0.00124201362405774, 0.0019440759412122286,
-        -0.02287937013013993, -0.002927608219628366, 0.018047571904430328, -0.01034426546939903;
+    input.jointPositions << -0.10922017141063572, 0.05081325960010118, 0.06581966291990003,
+        -0.0898053099824925, -0.09324922528169599, -0.05110058859172172, -0.11021232812838086,
+        0.054291515925228385, 0.0735575862560208, -0.09509332143185895, -0.09833823347493076,
+        -0.05367281245082792, 0.1531558711397399, -0.001030634273454133, 0.0006584764419034815,
+        -0.0016821925351926288, -0.004284529460797688, 0.030389771690123243, -0.040592118429752494,
+        -0.1695472679986807, -0.20799422095574033, 0.045397975984119654, -0.03946672931050908,
+        -0.16795588539580256, -0.20911090583076936, 0.0419854257806720;
+
+    input.jointVelocities = Eigen::VectorXd::Zero(26);
```

Comment:
Does it work only with zero joint velocities?

Reply:
no 😃 if you prefer, I can pass a random value for the joint state (position and velocity)

Reply:
I see. You can also keep the previous values (i.e. random but fixed values)

Reply:
Done here: 7c582cf18